### DISCUSSION TITLE
Librería replay buffer

### DISCUSSION THREAD

**Discussion by inveterate-cladocerus** [JavierBZ/Tarea3-IIC3697#26] [Q&A]
Hola, estoy usando una librería especializada como base para ReplayMemory,  ¿Puedo hacer esto o debo implementar esta clase de cero?
Saludos.

**Comment by JavierBZ** [comment]
Hola, si puedes usarla, pero debes explicar en detalle que hace cada función del ReplayMemory que estas importando.
